.github/workflows/end2end_test.yml

```diff
@@ -50,12 +50,12 @@ jobs:
             python-build-${{
               hashFiles(
                 '.github/workflows/end2end_test.yml',
-                'lightly_studio/e2e-tests/**',
-                'lightly_studio/src/**',
+                'lightly_studio/e2e-tests/**/*',
+                'lightly_studio/src/**/*',
                 'lightly_studio/pyproject.toml',
-                'lightly_studio_view/e2e/**',
-                'lightly_studio_view/src/**',
-                'lightly_studio_view/public/**',
+                'lightly_studio_view/e2e/**/*',
+                'lightly_studio_view/src/**/*',
+                'lightly_studio_view/public/**/*',
```

Comment:
I had an issue with the cache being hit even though there were changes. Added a star to be sure.